proto/lavanet/lava/spec/spec.proto

```diff
@@ -32,4 +32,6 @@ message Spec {
   ProvidersTypes providers_types = 14;
   repeated string imports = 15;
   repeated ApiCollection api_collections = 16;
+  repeated string contributor =17;
+  float contributor_percentage=18;
```

Comment:
i wouldnt use floats as it might not be deterministic with different systems. that is wht we have the sdk.dec, can we use that? or int